packages/neuron-ui/src/components/PageContainer/index.tsx

```diff
@@ -68,7 +68,7 @@ const PageContainer: React.FC<ComponentProps> = props => {
     return [
       bestBlockNumber > 0 && cacheTipBlockNumber > 0
         ? `${+((cacheTipBlockNumber * 100) / bestBlockNumber).toFixed(2)}%`
-        : '0',
+        : '0.00%',
```

Comment:
I think `0%` is clear and neat, but just my personal opinion

Reply:
I want to keep unify with `toFixed(2)`.